README.md

```diff
@@ -84,6 +84,17 @@ Optional settings:
 
 Ariadne Codegen implements a plugin system that enables further customization and fine-tuning of generated Python code. It’s documentation is available separately in the [PLUGINS.md](https://github.com/mirumee/ariadne-codegen/blob/main/PLUGINS.md) file.
 
+### Standard plugins
+
+Ariadne Codegen ships with optional plugins importable from the
+`ariadne_codegen.contrib` package:
+
+* [`ariadne_codegen.contrib.shorter_results.ShorterResultsPlugin`](ariadne_codegen/contrib/shorter_results.py)
+  \- This plugin processes generated client methods for operations where only
```

Comment:
```suggestion
  \- This plugin processes generated client methods for operations where only
```

Please don't do machinations like that in markdown. ;)

Reply:
Do you mean escaping the `-`? I removed the wrapping so no longer escaped